src/main/java/com/doyak/reflector/payload/exception/ExceptionAdvice.java

```diff
@@ -0,0 +1,89 @@
+package com.doyak.reflector.payload.exception;
+
+import java.util.UUID;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.lang.Nullable;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+import org.springframework.web.context.request.WebRequest;
+
+import com.doyak.reflector.payload.ApiResponse;
+import com.doyak.reflector.payload.code.status.ErrorStatus;
+import com.doyak.reflector.payload.dto.ErrorReasonDTO;
+
+
+@RestControllerAdvice
+public class ExceptionAdvice {
+
+    private static final Logger log = LoggerFactory.getLogger(ExceptionAdvice.class);
+
```

Comment:
```suggestion
@Slf4j 
@RestControllerAdvice
public class ExceptionAdvice {

```
프로젝트에 이미 Lombok을 사용하고 있으므로 일관성과 코드 간결성을 위해 직접 logger를 선언하는 대신 자동 로그 생성 어노테이션 `@Slf4j`를사용하는 것도 좋을 것 같아요. 하지만 라이브러리 의존성을 낮추고 싶다면 현재 방식도 좋습니다!

Reply:
이미 Lombok 사용하고 있으니까 그게 더 좋을 것 같네요!! 수정하겠습니다